### PR TITLE
Add Exercise 8

### PR DESCRIPTION
**Pull request by lubos-harasta**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (anishnaik) [inline .vscode/settings.json:1]: @lubos-harasta can you `git rm` this file from your stage?

**Comment by anishnaik** [comment]
@lubos-harasta thank you for the submission! I will check this out in the coming days.

**Review by glarregay-tob — CHANGES_REQUESTED** [review]
This commit deletes files and adds entries to .gitignore.
Can you please delete this commit, keeping only the changes made to Excercise-8.md?

**Comment by lubos-harasta** [comment]
> This commit deletes files and adds entries to .gitignore. Can you please delete this commit, keeping only the changes made to Excercise-8.md?

@glarregay-tob Done (hopefully in the correct way).

**Pull request merged**